src/centrality.rs

```diff
@@ -0,0 +1,217 @@
+use pyo3::prelude::*;
+
+use petgraph::graph::NodeIndex;
+use petgraph::visit::{
+    GraphBase,
+    GraphProp, // allows is_directed
+    IntoNeighborsDirected,
+    IntoNodeIdentifiers,
+    NodeCount,
+    NodeIndexable,
+};
+use std::collections::HashMap;
```

Comment:
We typically use the [Hashbrown](https://github.com/rust-lang/hashbrown/) HashMap type instead of the one from stdlib, mostly because it is faster out of the box because it uses a different hasher which is not DoS resistant and also adds support for multithreaded iterators via rayon. It's a drop in replacement (and actually the upstream of the stdlib implementation), so just:
```suggestion
use hashbrown::HashMap;
```